user/src/com/google/gwt/user/client/ui/RootPanel.java

```diff
@@ -223,16 +232,36 @@ public static native com.google.gwt.user.client.Element getBodyElement() /*-{
 
   /**
    * Determines whether the given widget is in the detach list.
-   * 
```

Comment:
Alternative: 

Remove added paragraph and then:

`@deprecated The detach list was used to prevent memory leaks in old browsers which is not relevant anymore. See deprecation notice in {@link #detachOnWindowClose} for further information.`